multitenancy/tenant-manager-datamodel/src/main/java/io/apicurio/multitenant/api/TenantsResource.java

```diff
@@ -30,7 +29,7 @@
   @POST
   @Produces("application/json")
   @Consumes("application/json")
-  Response createTenant(NewRegistryTenantRequest data);
+  RegistryTenant createTenant(NewRegistryTenantRequest data);
```

Comment:
I'll manually change this back

Reply:
@EricWittman FYI , in case you know if it's possible to force Apicurio Studio to use `Response`

Reply:
If not, we can add support.  :)  I'll check.

Reply:
Not yet supported, but here is where we would add that functionality:  https://github.com/Apicurio/apicurio-studio/blob/master/back-end/hub-codegen/src/main/java/io/apicurio/hub/api/codegen/jaxrs/OpenApi2CodegenVisitor.java#L305-L344

Feel free to either 1) submit a PR or 2) open an issue and I'll knock it out